applications/CompressiblePotentialFlowApplication/tests/cpp_tests/test_incompressible_potential_element_3d.cpp

```diff
@@ -0,0 +1,232 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Inigo Lopez
```

Comment:
These are basically a copy paste from your tests so I keep you as author of this file @inigolcanalejo

Reply:
Sounds good, feel free to add you name too if you want, since you are also involved in the development!